stripe/api_requestor.py

```diff
@@ -252,14 +252,21 @@ def request_raw(self, method, url, params=None, supplied_headers=None):
             for key, value in supplied_headers.items():
                 headers[key] = value
 
+        util.log_info('Stripe {method} request to {path}',
+                      method=method, path=abs_url)
```

Comment:
Ah! I remember you mentioning this before, but I couldn't find anything on it. This is totally doable, I'll add it.